src/main/java/org/jenkinsci/plugins/badge/BadgeAction.java

```diff
@@ -5,36 +5,67 @@
 import jenkins.model.Jenkins;
 
 import org.kohsuke.stapler.HttpResponse;
-import org.kohsuke.stapler.QueryParameter;
-
 
 /**
-* @author Kohsuke Kawaguchi
-*/
+ * @author Kohsuke Kawaguchi
+ */
 public class BadgeAction implements Action {
+
+    /**
+     * TO DO
+     */
     private final BadgeActionFactory factory;
 
+    /**
+     * TO DO
+     */
     public final Job project;
 
+    /**
+     * TO DO
+     *
+     * @param factory
+     * @param project
+     */
     public BadgeAction(BadgeActionFactory factory, Job project) {
         this.factory = factory;
         this.project = project;
     }
 
+    /**
+     * TO DO
+     *
+     * @return
+     */
+    @Override
     public String getIconFileName() {
-        return Jenkins.RESOURCE_PATH+"/plugin/embeddable-badges/images/24x24/shield.png";
+        return Jenkins.RESOURCE_PATH + "/plugin/embeddable-badges/images/24x24/shield.png";
     }
 
+    /**
+     * TO DO
+     *
+     * @return
+     */
+    @Override
     public String getDisplayName() {
         return Messages.BadgeAction_DisplayName();
     }
 
+    /**
+     * TO DO
+     *
+     * @return
+     */
+    @Override
```

Comment:
[Checkstyle] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2.